omnibus/config/software/system-probe.rb

```diff
@@ -15,9 +15,12 @@
   mkdir "#{install_dir}/embedded/share/system-probe/ebpf/runtime"
   mkdir "#{install_dir}/embedded/share/system-probe/ebpf/co-re"
   mkdir "#{install_dir}/embedded/share/system-probe/ebpf/co-re/btf"
+  mkdir "#{install_dir}/embedded/share/system-probe/java"
   mkdir "#{install_dir}/embedded/nikos/embedded/bin"
   mkdir "#{install_dir}/embedded/nikos/embedded/lib"
 
+  copy 'pkg/network/java/agent-usm-7.43.0.jar', "#{install_dir}/embedded/share/system-probe/java/"
```

Comment:
is `7.43.0` hard coded by intention? will it remain `7.43.0` forever?

Reply:
we need to commit a binary on each upgrade of the java tracer, meanwhile we didn't define versioning.
But probably using the dd-trace-java versioning would we better, or no versioning ...

Reply:
@val06

Reply:
we will use a dd-java-agent official version (the java agent is release in different cycles rather than dd-agent)
if we want to upgrade the dd-java-agent payload, we will need to update this deployment file